README.md

```diff
@@ -2,3 +2,12 @@ fluentd-catch-all-config
 ========================
 
 Collection of configuration files for the Fluentd log collection agent. Intended to enable the automagic collection of most logs generated by many of the most popular applications run in the cloud.
+
+Major version updates may introduce changes that are not backwards compatible.
+
+This package attempts to keep parity with the [Fluentd Parsers](https://docs.fluentd.org/v0.12/articles/parser-plugin-overview) when providing structured logs. The parsers that this package currently uses are:
+
+* apache2
```

Comment:
Minor: while we're at it, let's also use code font for these names, as they're literal parser names.